pytorch/docker-compose.yaml

```diff
@@ -165,16 +165,42 @@ services:
         dependency.apt.openjdk-17-jdk: true
         dependency.idp: false
         dependency.python.ipex: requirements.txt
-        dependency.python.pip: torchserve-requirements.txt
+        dependency.python.pip: serving/torchserve-requirements.txt
         docs: serving
         org.opencontainers.base.name: "intel/python:3.10-core"
         org.opencontainers.image.title: "Intel® Extension for PyTorch Serving Image"
         org.opencontainers.image.version: ${IPEX_VERSION:-2.4.0}-serving-cpu
-      target: torchserve
+      target: torchserve-cpu
     command: torchserve --version
     entrypoint: ""
     extends: ipex-base
-    image: ${REGISTRY}/${REPO}:b-${GITHUB_RUN_NUMBER:-0}-${BASE_IMAGE_NAME:-ubuntu}-${BASE_IMAGE_TAG:-22.04}-py${PYTHON_VERSION:-3.10}-torchserve
+    image: ${REGISTRY}/${REPO}:b-${GITHUB_RUN_NUMBER:-0}-${BASE_IMAGE_NAME:-ubuntu}-${BASE_IMAGE_TAG:-22.04}-py${PYTHON_VERSION:-3.10}-torchserve-cpu
+    ports:
+      - 8080:8080
+      - 8081:8081
+      - 8082:8082
+      - 7070:7070
+      - 7071:7071
+  torchserve-xpu:
+    build:
+      args:
+        PACKAGE_OPTION: pip
+        XPU_SMI_VERSION: ${XPU_SMI_VERSION:-1.2.38}
+        TORCHSERVE_BASE: ipex-xpu-base
+      labels:
```

Comment:
I think you're missing a few dependencies like xpu-smi

Reply:
maybe we can focus on docs + labels in a followup PR. I don't want this one hanging for weeks.

Reply:
added version to the labels section of compose